### PR TITLE
net-misc/loki-promtail-bin add ebuild

### PR DESCRIPTION
**Pull request by ibra-adjust**
Add an ebuild for net-misc/loki-promtail-bin

### CODE REVIEW

File: net-misc/loki-promtail-bin/files/promtail.yml
Comment (sibuthomasmathew) [inline net-misc/loki-promtail-bin/files/promtail.yml:1]: The configuration file must be kept independent of the build

Comment (ibra-adjust) [reply net-misc/loki-promtail-bin/files/promtail.yml:1]: Could you please explain why it must be kept independent of the build ?

Comment (sibuthomasmathew) [reply net-misc/loki-promtail-bin/files/promtail.yml:1]: A lot of the configuration like file `positions` or `client.url` can change. So, we need to flexibility to change it dynamically using ansible or other options.

File: net-misc/loki-promtail-bin/files/promtail.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: 'http://localhost:3100/loki/api/v1/push'
```
Comment (sibuthomasmathew) [inline net-misc/loki-promtail-bin/files/promtail.yml:9]: Is this a dummy value?

File: net-misc/loki-promtail-bin/files/promtail.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+scrape_configs:
+  - job_name: system
+    static_configs:
+      - targets:
+          - localhost
+        labels:
+          job: varlogs
+          __path__: /var/log/*log
```
Comment (sibuthomasmathew) [inline net-misc/loki-promtail-bin/files/promtail.yml:18]: Do we want promtail reading all the log files?

**Review by sibuthomasmathew — CHANGES_REQUESTED** [review]

File: net-misc/loki-promtail-bin/files/promtail.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
```
Comment (sibuthomasmathew) [inline net-misc/loki-promtail-bin/files/promtail.yml:3]: We don't need promtail running as a server. Let's disable it
```suggestion
 disable: true
```

File: net-misc/loki-promtail-bin/files/promtail.yml
Comment (ibra-adjust) [reply net-misc/loki-promtail-bin/files/promtail.yml:1]: OK, I will remove the configuration file.

**Review by sibuthomasmathew — APPROVED** [review]

**Pull request closed without merging**